betamax/options.py

```diff
@@ -17,10 +21,8 @@ def validate_serializer(serializer):
 
 
 def validate_placeholders(placeholders):
-    keys = ['placeholder', 'replace']
-    return all(
-        sorted(list(p.keys())) == keys for p in placeholders
-    )
+    """Validate placeholders is a dict-like structure"""
+    return hasattr(placeholders, 'items')
```

Comment:
An empty dictionary is a valid set of placeholders. The previous code also passed on an empty list.